packages/shared/terrazzo.config.js

```diff
@@ -1,40 +0,0 @@
-import { defineConfig } from '@terrazzo/cli';
-import css from '@terrazzo/plugin-css';
-
-function pxToRem(token) {
-	if (token.$type === 'dimension' && token.$value.slice(-2) === 'px') {
-		return token.$value.slice(0, -2) / 16 + 'rem';
-	}
-}
-
-function clearFxPrefix(id) {
-	if (id.includes('fx.')) {
-		return id.replace('fx.', '').replace('.', '-');
-	}
-}
```

Comment:
Why is this getting removed? We have styled components in the shared package. Don't they need the pxToRem thing?

Reply:
[terrazzo.config.js](https://github.com/gitbutlerapp/gitbutler/pull/5878/files/1ad7bf4c65d71fd4a82e657a3db2bf35ac16c495#diff-fc7a96aab239a09573e1eda540720c8e283247a2d95ec9b2712b90f91fc256f1) should only convert design tokens, and this is needed only in the UI package